### DISCUSSION TITLE
Attivare Routine Alexa da Home Assistant

### DISCUSSION THREAD

**Discussion by alessandrobonaldi** [indomus/forum#3017] [HUB: Home Assistant]
Ciao a tutti.
qualcuno sa spiegarmi come posso scatenare una routine costruita tramite Alexa da Home Assistant?
in pratica vorrei creare dei pulsanti virtuali che una volta premuti facciano partire una determinata routine.
è possibile?

grazie!
Ale

**Comment by lux90ita** [comment]
è integrato con haaska?
comunque devi provare a modificare yaml

alexa:
  smart_home:
    filter:
      exclude_domains:
        - alarm_control_panel
        
     include_entities:
        #includere progressivamente, in questa sezione, le entità da esporre ad Alexa
      include_domains:
        #includere progressivamente, in questa sezione, i domini da esporre ad Alexa

**Reply by indomus** [reply]
leggi bene, sta chiedendo una cosa diversa (che non si può fare, per inciso).

**Reply by lux90ita** [reply]
Questo non può fare al caso suo?
Fonte
link1
https://community.home-assistant.io/t/how-to-trigger-alexa-routines-from-home-assistant/90678/149
link2
https://community.home-assistant.io/t/how-to-control-amazon-smart-plugs-from-home-assistant-via-emulated-hue-and-alexa-media-player/464254
